tfhe/src/shortint/backward_compatibility/server_key/mod.rs

```diff
@@ -19,14 +20,15 @@ impl<'vers, C: Container<Element = concrete_fft::c64>>
 }
 
 #[derive(Serialize, Deserialize)]
+#[cfg_attr(tfhe_lints, allow(tfhe_lints::serialize_without_versionize))]
```

Comment:
should this maybe a global cfg attr for all backwad_compatibility modules ? or do we expect to have some serialize + Versionize in these modules ?

Reply:
yes good idea !

Reply:
ok, did your investigation for the allow stuff indicate anything to fix it or is it a weird bug we can't really do anything about ?

Reply:
I think I might have a problem with the way the lint is declared, since clippy and marker do not seem to check this inside every lint. But I did not find any obvious difference (on the surface) between what I do and what marker is doing.

Reply:
But marker and clippy are adding their own api on top of the rustc one.